cmd/cli/operations/update_job.go

```diff
@@ -24,9 +24,9 @@ type UpdateJob struct {
 	AllowNonRestoredState bool
 }
 
-func (o RealOperator) filterRunningJobs(jobs []flink.Job) (ret []flink.Job) {
+func (o RealOperator) filterRunningJobsByName(jobs []flink.Job, jobNameBase string) (ret []flink.Job) {
 	for _, job := range jobs {
-		if job.Status == "RUNNING" {
+		if job.Status == "RUNNING" && job.Name == jobNameBase {
```

Comment:
Since we're talking about the job name base it should check the start of the `job.Name` instead of equaling completely. Something like `strings.HasPrefix(job.Name, jobNameBase)`?

Reply:
Done.